src/ex4/server/db/migrations/20220621210940-create-items.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Items', {
+      item_id: {
```

Comment:
You can use id and name. The name of the table is already items so it is clear that each row is an item.